django/db/backends/sqlite3/operations.py

```diff
@@ -217,6 +217,10 @@ def convert_datetimefield_value(self, value, expression, connection, context):
         if value is not None:
             if not isinstance(value, datetime.datetime):
                 value = parse_datetime(value)
+
+            if timezone.is_aware(value):
+                return value
+
             if settings.USE_TZ:
```

Comment:
I feel like `if settings.USE_TZ and not timezone.is_aware(value)` may be simpler rather than adding another place that the function returns from.